### ISSUE TITLE
delayStart/delayStop for directive

### ISSUE THREAD

**Issue by kopach** [kuuurt13/ng-block-ui#44]
Is there any way to use `delayStart/delayStop` with `*blockUI` directive?

**Comment by kuuurt13** [comment]
You should be able to but after looking into it I found a bug. For some reason angular makes you namespace all other parameters for structural directives. I just need to namespace these inputs and it should work. I can try to release a fix by tomorrow morning.

**Comment by kuuurt13** [comment]
@kopach I got out of work early so I had time to fix this tonight. I released a new npm package with the fix, please update to `1.0.0-beta.16` for Angular 4.x and `0.9.3` for Angular 2.x. To pass multiple parameters to structural directives Angular makes you pass them like below
```html
<div *blockUI="'hero-section'; delayStart:2000; delayStop:2000;"></div>
```

Also, here is a working [plunker](https://embed.plnkr.co/ezOdUwK2d4TRrBCNsAoK?show=app.compontent.html,preview&autoCloseSidebar) as well. Closing this but if you have any issues, feel free to open it again.
